test/sudoku.fs

```diff
@@ -0,0 +1,94 @@
+module sudoku
+
+open System.Collections.Generic
+
+type Box = int
+type Sudoku = Box array array
+
+let rows = id
+let cols (sudoku:Sudoku) =
+    sudoku
+    |> Array.mapi (fun a row -> row |> Array.mapi (fun b cell -> sudoku.[b].[a]))
+
+let getBoxIndex count row col = 
+   let n = row/count
+   let m = col/count
+   n * count + m
+
+let boxes (sudoku:Sudoku) = 
+    let d = sudoku |> Array.length |> float |> System.Math.Sqrt |> int
```

Comment:
instead of 'System.Math.Sqrt` we can implement `sqrt` in `Microsoft.FSharp.Core.Operators.erl` (or whereever that lives in FSharp.Core).